src/js/hooks/list-pages/purchase-order/usePurchaseOrderListTableData.jsx

```diff
@@ -35,7 +35,7 @@ const usePurchaseOrderListTableData = (filterParams) => {
         max: `${state.pageSize}`,
         ...sortingParams,
         ...filterParams,
-        paymentTerm: paymentTerm && paymentTerm.map(term => term.id),
+        paymentTerm: paymentTerm && paymentTerm.map(({ id }) => id),
```

Comment:
just FYI:
could be shorten to:
```suggestion
        paymentTerm: paymentTerm.map?.(({ id }) => id),
```